components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -0,0 +1,258 @@
+import Container from '../../Container';
+import { Flex } from '../../Grid';
+import Loading from '../../Loading';
+import { P, Span } from '../../Text';
+
+const FilterLabel = styled.label`
+  font-weight: 500;
+  text-transform: uppercase;
+  margin-bottom: 8px;
+  color: #4e5052;
+`;
+
+const FundAmounts = styled.div`
+  width: 100%;
+  height: 48px;
+  border-radius: 10px;
+  background-color: #29cc75;
+  border-right: 360px solid #e03f6a;
+  padding-top: 10px;
+  padding-left: 5px;
+  @media (max-width: 1025px) {
+    height: 130px;
+    background-color: #29cc75;
+    border-right: 0px;
+    border-bottom: 40px solid #e03f6a;
+  }
+`;
+
+const TotalFundsLabel = styled(Container)`
+  display: table-cell;
+  padding-left: 10px;
+  height: 26px;
+  border-radius: 5px;
+  background-color: white;
+  opacity: 80%;
+  vertical-align: middle;
+`;
+
+const Square = styled(Container)`
+  width: 8px;
+  height: 8px;
+  display: inline-block;
+  background-color: ${props => props.color};
+`;
+
+const transactionsOverviewQuery = gqlV2/* GraphQL */ `
+  query TransactionsOverviewQuery(
+    $hostSlug: String!
+    $accounts: [AccountReferenceInput!]
+    $dateFrom: String
+    $dateTo: String
+  ) {
+    host(slug: $hostSlug) {
+      id
+      contributionStats(accounts: $accounts, from: $dateFrom, to: $dateTo) {
+        numContributions
+        numOneTime
+        numRecurring
+        dailyAvgIncome
+      }
+      expenseStats(accounts: $accounts, from: $dateFrom, to: $dateTo) {
+        numExpenses
+        dailyAverage
+        numInvoices
+        numReimbursements
+        numGrants
+      }
+    }
+  }
+`;
+
+const TransactionsOverviewSection = ({ hostSlug, currency }) => {
+  const [dateFrom, setDateFrom] = useState(null);
+  const [dateTo, setDateTo] = useState(null);
+  const [collectives, setCollectives] = useState(null);
+
+  const { data, loading, refetch } = useQuery(transactionsOverviewQuery, {
+    variables: { hostSlug, dateFrom, dateTo, accounts: collectives },
+    context: API_V2_CONTEXT,
+  });
+  const contributionStats = data?.host.contributionStats;
+  const expenseStats = data?.host.expenseStats;
+
+  useEffect(() => {
+    refetch();
+  }, [dateFrom, dateTo, collectives]);
+
+  const { numContributions, numRecurring, numOneTime, dailyAvgIncome } = contributionStats || 0;
+  const { numExpenses, numInvoices, numReimbursements, numGrants, dailyAverage } = expenseStats || 0;
+
+  const setDate = dateRange => {
+    if (!dateRange) {
+      setDateFrom(null);
+      setDateTo(null);
+      refetch();
+      return;
+    }
+    const dates = dateRange.split('→');
+    const dateFrom = dates[0];
+    const dateTo = dates[1];
+    if (dateFrom === 'all') {
+      setDateFrom(null);
+    } else {
+      setDateFrom(dateFrom);
+    }
+    if (dateTo === 'all') {
+      setDateTo(null);
+    } else {
+      setDateTo(dateTo);
+    }
+  };
+
+  const getDateString = () => {
```

Comment:
We'll need to export it, but this function can be replaced by a call to `encodePeriod`.